doc/examples/filters/plot_deconvolution.py

```diff
@@ -26,15 +26,23 @@
 
 rng = np.random.default_rng()
 
+# Convert astronaut's image to grayscale
 astro = color.rgb2gray(data.astronaut())
 
+# Define the Point Spread Function (PSF)
 psf = np.ones((5, 5)) / 25
-astro = conv2(astro, psf, 'same')
-# Add Noise to Image
-astro_noisy = astro.copy()
-astro_noisy += (rng.poisson(lam=25, size=astro.shape) - 10) / 255.0
 
-# Restore Image using Richardson-Lucy algorithm
+# Convolve image with the PSF to simulate a blurred image
+astro_blurred = conv2(astro, psf, 'same')
+
+# Introduce poisson noise to the blurred image
```

Comment:
Can we provide a link here to a page that describes why this type of noise commonly occurs?

Reply:
I couldn't find a source using poisson noise but [here](https://www.mathworks.com/help/images/deblurring-images-using-the-lucy-richardson-algorithm.html) they have used gaussian noise. So, pls LMK if you think it's ok to reference mathworks in scikit-image and then I can change it gaussian noise instead. Thanks!

Reply:
The "Richardson Lucy deconvolution" wikipedia page is already a reference. I don't think we should put that in the comment again. And I think this (https://en.wikipedia.org/wiki/Image_noise) might be a better reference to put here as it aligns better with what @stefanv asked for in the above comment. LMKWYT.